src/verilog/verilog_expr.h

```diff
@@ -18,7 +18,7 @@ Author: Daniel Kroening, kroening@kroening.com
 class hierarchical_identifier_exprt : public binary_exprt
 {
 public:
-  hierarchical_identifier_exprt() : binary_exprt(ID_hierarchical_identifier)
+  hierarchical_identifier_exprt() : binary_exprt(exprt(), ID_hierarchical_identifier, exprt())
```

Comment:
This doesn't look like a desirable way to construct expressions?

Reply:
I'll simply remove these for now.